gcc/config/aarch64/aarch64-protos.h

```diff
@@ -741,6 +741,28 @@ class aarch64_simd_switcher
   bool m_old_general_regs_only;
 };
 
+/* SEH support */
```

Comment:
I've moved the `extern` definitions here as similar ones can be found in `i386-protos.h`. I am not sure about whether the current position is the correct one though and why there is only few such functions defined for aarch64 while i386 have much more defined?